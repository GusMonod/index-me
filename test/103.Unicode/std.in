úƨúáℓ ïñƥúƭ λáƨ ₥úℓƭïƥℓè ℓïñèƨ ƭλáƭ ářè ƒáïřℓ¥ ℓôñϱ, ωïƭλ ωôřδ řèƥèáƭ áñδ ƨúçλ ƭλïñϱƨ ƭλáƭ ářè ƒáïřℓ¥ èáƨ¥ ƭô ωřïƭè ïñ ƭèƨƭ ƒïℓèƨ
áñδ λèřè ïƨ ƭλè ñèжƭ δôçú₥èñƭ, ωïƭλ ñáƭúřáℓ ℓáñϱúáϱè ôñçè áϱáïñ Ì λáƲè ñô ï₥áϱïñáƭïôñ áƨ ¥ôú çáñ řèáℓïƺè
ƒïñáℓℓ¥, λèřè ïƨ ƭλè ℓáƨƭ ôƒ ƭλè δôçú₥èñƭƨ Ì'₥ ϱôïñϱ ƭô ωřïƭè ƒôř ƭλïƨ ƭèƨƭ ƨúïƭè ;-)
