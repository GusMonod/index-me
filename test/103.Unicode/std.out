úƨúáℓ: 0
ïñƥúƭ: 0
λáƨ: 0
₥úℓƭïƥℓè: 0
ℓïñèƨ: 0
ƭλáƭ: 0
ářè: 0
ƒáïřℓ¥: 0
ℓôñϱ,: 0
ωïƭλ: 0, 1
ωôřδ: 0
řèƥèáƭ: 0
áñδ: 0, 1
ƨúçλ: 0
ƭλïñϱƨ: 0
èáƨ¥: 0
ƭô: 0, 2
ωřïƭè: 0, 2
ïñ: 0
ƭèƨƭ: 0, 2
ƒïℓèƨ: 0
λèřè: 1, 2
ïƨ: 1, 2
ƭλè: 1, 2
ñèжƭ: 1
δôçú₥èñƭ,: 1
ñáƭúřáℓ: 1
ℓáñϱúáϱè: 1
ôñçè: 1
áϱáïñ: 1
Ì: 1
λáƲè: 1
ñô: 1
ï₥áϱïñáƭïôñ: 1
áƨ: 1
¥ôú: 1
çáñ: 1
řèáℓïƺè: 1
ƒïñáℓℓ¥,: 2
ℓáƨƭ: 2
ôƒ: 2
δôçú₥èñƭƨ: 2
Ì'₥: 2
ϱôïñϱ: 2
ƒôř: 2
ƭλïƨ: 2
ƨúïƭè: 2
;-): 2